README.md

```diff
@@ -1,8 +1,62 @@
-This plugin does a HTTP/HTTPS request to a speficied URL with the build parameters automatically
-part of the payload as key/value pairs.
+# Http Request Plugin for Jenkins
 
-It can be configured whether GET or POST (default) should be used as HTTP method, as well
-as if the status return code of the response marks the build as failed (default) or is ignored.
-You can specify different username / password pairs in the global configuration to support
-authenticated HTTP/HTTPS requests, which you then refer to from your build configuration.
+This plugin sends a HTTP/HTTPS request to a user speficied URL.
 
+## Features
+
+The following features are available in both the non-Pipeline and in
+the Pipeline version of the plugin.
```

Comment:
Well, there is no such Pipeline version. I would say: _The following features are available in both Pipeline and traditional project types_